WordPress/build.gradle

```diff
@@ -35,8 +35,8 @@ android {
         minSdkVersion 14
         targetSdkVersion 23
 
+        vectorDrawables.useSupportLibrary = true
```

Comment:
Yep revert this please - ref https://github.com/wordpress-mobile/WordPress-Android/issues/3791